flink-runtime-web/web-dashboard/app/scripts/common/filters.coffee

```diff
@@ -28,6 +28,32 @@ angular.module('flinkApp')
 
   amDurationFormatExtendedFilter
 
+.filter "humanizeDuration", ->
```

Comment:
Fair enough. But since the functionality required here is little bit unusual [two types of strings], it made sense to write the code instead of using even a direct `moment.duration()` function call.